spec/decorator_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'spec_helper'
+
+describe Decorator do
+  before do
+    @nameable = double('Nameable', correct_name: 'John Smith')
+    @decorator = Decorator.new(@nameable)
+  end
+
+  describe '#correct_name' do
+    it 'returns the correct name' do
+      expect(@decorator.correct_name).to eq('John Smith')
+    end
+  end
+end
+
+describe CapitalizeDecorator do
```

Comment:
- [ ] [optional] It would be nice if each class had its own tests file

Reply:
- [ ] @team you forgot to add tests for the nameable class, kindly add the tests